src/components/app/pageUserProfile/index.tsx

```diff
@@ -158,6 +171,20 @@ export function PageUserProfile({ params, searchParams, user }: PageUserProfile)
           performedUserActionTypes={performedUserActionTypes}
         />
       </section>
+      {userActionsFromPreviousCampaigns.length > 0 && (
+        <section>
+          <PageTitle className="mb-4" size="sm">
+            Previous campaigns
+          </PageTitle>
+          <PageSubTitle className="mb-5">
+            Nice work. You completed {userActionsFromPreviousCampaigns.length} previous{' '}
+            {userActionsFromPreviousCampaigns.length > 1 ? 'campaigns' : 'campaign'}.
```

Comment:
np: we have a `pluralize` helper for this
```suggestion
            {pluralize({ singular: 'campaign', count: userActionsFromPreviousCampaigns.length }}.
```

Reply:
Ty! Didn't know that we had this function